graylog2-server/src/main/java/org/graylog2/lookup/adapters/HTTPJSONPathDataAdapter.java

```diff
@@ -284,7 +284,7 @@ public Descriptor() {
         public Config defaultConfiguration() {
             return Config.builder()
                     .type(NAME)
-                    .url("")
+                    .url("https://example.com/api/lookup?key=${key}")
```

Comment:
I think we should not use a valid domain for the initial value, to avoid leaking something when a user accidentally uses the default.